docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,195 @@
+= Getting Started with the Hazelcast Python Client
+:page-layout: tutorial
+:page-product: imdg
+:page-categories: Caching, Getting Started
+:page-lang: python3
+:page-est-time: 5-10 mins
+:description: This tutorial will get you started with the Hazelcast Python client.
+
+== What You'll Learn
+
+{description}
+
+== Before you Begin
+
+* A text editor or IDE
+* Docker
+* Python 3.6+
+
+== Start a Hazelcast Member
+
+We will use the 5.3 version of Hazelcast for this tutorial.
```

Comment:
why not say use the `latest` version of Hazelcast so that the tutorial is up to date all the time. For this tutorial, that will what is latest at the Viridian.

Reply:
All our guides have the version specified - this is a consistency copy from C# and Java.

Suggest fixing them ALL after the first cut is made available.

Reply:
agree, and that is why they left on older version.